### DISCUSSION TITLE
URL imports

### DISCUSSION THREAD

**Discussion by brundonsmith** [oven-sh/bun#4753] [Ideas]
One of my favorite features of Deno is the ability to import a module directly from a URL. This is also supported by ES modules in the browser, so it seems natural to support it in Bun as well

Apologies if this feature has already been discussed/is already on the roadmap!

**Comment by sirenkovladd** [comment]
https://github.com/oven-sh/bun/blob/bfaf095c2ed2ba1f0cd35f1658e2babee8b51985/test/js/bun/plugin/plugins.test.ts#L16-L34

**Reply by brundonsmith** [reply]
Very cool that it's so easy to implement as a plugin! Still feels like something that makes sense to build-in though

**Reply by sirenkovladd** [reply]
You may be right
https://github.com/oven-sh/bun/issues/38

**Reply by brundonsmith** [reply]
Oh nice, I missed that ticket when I was looking before opening this discussion. Disregard!
